src/ctia/lib/map.clj

```diff
@@ -5,3 +5,12 @@
   (if (every? map? vals)
     (apply merge-with rmerge vals)
     (last vals)))
+
+(defn dissoc-in
+  [m [k & ks :as keys]]
+  (if ks
+    (if-let [nextmap (get m k)]
+      (let [newmap (dissoc-in nextmap ks)]
+        (assoc m k newmap))
+      m)
+    (dissoc m k)))
```

Comment:
This is very manual. I'd rather see it based on update-in. e.g.:

(defn dissoc-in
  [m keys]
  (let [ks (butlast keys)
        lk (last keys)]
    (update-in m ks dissoc lk)))